packages/server/src/server.ts

```diff
@@ -288,7 +288,7 @@ export class Server extends EventEmitter {
       abortWithDelayController.signal.removeEventListener('abort', onDelayedAbort)
       setTimeout(() => {
         requestAbortController.abort(err)
-      }, 3000)
+      }, this.options.lockDrainTimeout || 3000)
```

Comment:
I would like to make it a habit to set default values in the `constructor`, then we'll have a single overview of all the default values.